### ISSUE TITLE
[FR] Extruder Filament Force Sensor

### ISSUE THREAD

**Issue by p2004r** [MarlinFirmware/Marlin#18521]
A long time ago, in a 3d_print_forum, far away…

http://airtripper.com/1338/airtripper-extruder-filament-force-sensor-introduction/

When the extruder feeds the filament to the hot end, the extruder is effectively pushing against the filament causing the extruder to apply extra load on the load cell. Load cells have strain gauges attached that change in electrical resistance when under different loads. This resistance change provides small voltage levels that can be amplified and then read by an analogue to digital converter. 

Basically, the purpose of doing these graphs is to find out what sort of information we can get out of the load cell that is used for the filament force sensor. We want to find out if the graphs can be used as a guide for better extruder set-ups. 

By connecting this sensor to Marlin, the retract distance, the maximum printing speed and the optimal nozzle temperature are automatically adjusted. Filament plugs, nozzle clogging, filament breakage and damage to the fitings are recognized.

**Comment by p2004r** [comment]
https://github.com/MarlinFirmware/Marlin/pull/2309  old HX711 based ABL code
